itest/starboard/starboard_cli_test.go

```diff
@@ -1014,10 +1014,29 @@ var _ = Describe("Starboard CLI", func() {
 
 	Describe("Command scan ciskubebenchreports", func() {
 
+		It("scan wrong node name. It doesn't create CISKubeBenchReports", func() {
+			err := cmd.Run(versionInfo, []string{
+				"starboard",
+				"scan", "ciskubebenchreports", "notExistNode",
+				"-v", starboardCLILogLevel,
+			}, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+
+			var nodeList corev1.NodeList
+			err = kubeClient.List(context.TODO(), &nodeList)
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, node := range nodeList.Items {
```

Comment:
I think that more reliable test would be to spin up 3 nodes KIND cluster, scan just two nodes, and then assert that we only have two CISKubeBenchReport objects.

Reply:
I updated to spin up 3 nodes, I updated with 2 test about cisKube-bench:
- should create a specific CISKubeBenchReports: starboard scan ciskubebenchreports kind-control-plane
- should create all CISKubeBenchReport: starboard scan ciskubebenchreports